x/ibc/02-client/client/cli/query.go

```diff
@@ -55,18 +54,16 @@ $ %s query ibc client states
 // a given id as defined in https://github.com/cosmos/ics/tree/master/spec/ics-002-client-semantics#query
 func GetCmdQueryClientState(clientCtx client.Context) *cobra.Command {
```

Comment:
Could remove `client.Context` arg from GetCmd* as part of this refactor too

Reply:
you need to pass in the clientCtx afaik. cc: @alexanderbez

Reply:
Actually @fedekunze we need to remove it. A `client.Context` should never be passed as an arg -- query or tx. I'm not sure how I missed this in `x/bank`. Thanks for catching this @amaurymartiny!

x/bank PR: https://github.com/cosmos/cosmos-sdk/pull/6645